src/tbb/CMakeLists.txt

```diff
@@ -181,6 +191,7 @@ if (TBB_INSTALL)
             COMPONENT devel)
 endif()
 
-if (COMMAND tbb_gen_vars)
+# not useful when bundled into a framework
+if (COMMAND tbb_gen_vars AND NOT TBB_BUILD_APPLE_FRAMEWORKS)
```

Comment:
Could you please elaborate a bit more in this comment, why vars scripts are not useful here?

Reply:
In hindsight, there is actually no harm/problem in doing this, but I do have to fix where the vars.sh file gets generated when using Apple frameworks, as the binary itself is located inside a framework, and the vars.sh file should be besides the framework bundle.
I've undone this change, and fixed the tbb_gen_vars macro to place the vars file correctly when using Apple frameworks (and that the variables in the vars.sh file don't point inside the tbb framework file structure).